src/lib/InstantSearch.ts

```diff
@@ -389,7 +389,7 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
     });
 
     if (this.routing) {
-      this._routingManager.applyStateFromRoute();
+      this._routingManager.applyStateFromRoute(this.routing.router.read());
```

Comment:
We already have read the URL at the creation of the RoutingManager. I don't think we have to read again from the router since the value is already inside the `initialUiState`. Thoughts?

Reply:
That would make sense to me to use `initialUiState`.